redbot/core/core_commands.py

```diff
@@ -1600,16 +1600,24 @@ async def leave(self, ctx: commands.Context, *servers: GuildConverter):
             return await ctx.send(_("You need to specify at least one server ID."))
 
         leaving_local_guild = not guilds
+        number = len(guilds)
 
         if leaving_local_guild:
             guilds = (ctx.guild,)
             msg = (
                 _("You haven't passed any server ID. Do you want me to leave this server?")
                 + " (y/n)"
             )
+        if number > 1:
+            msg = (
+                _(f"Are you sure you want me to leave these servers?")
+                + " (y/n):\n"
+                + "\n".join(f"- {guild.name} (`{guild.id}`)" for guild in guilds)
+            )
+
         else:
             msg = (
-                _("Are you sure you want me to leave these servers?")
+                _("Are you sure you want me to leave this server")
                 + " (y/n):\n"
                 + "\n".join(f"- {guild.name} (`{guild.id}`)" for guild in guilds)
```

Comment:
This can be simplified considering we're only dealing with a single guild:
```suggestion
                + f"- {guilds[0].name} (`{guilds[0].id}`)"
```